assets/keymaps/vim.json

```diff
@@ -215,7 +215,7 @@
       "shift-d": "vim::DeleteToEndOfLine",
       "shift-j": "vim::JoinLines",
       "y": ["vim::PushOperator", "Yank"],
-      "shift-y": "vim::YankLine",
```

Comment:
This is the only use of `vim:: YankLine`. I didn't delete that function because I thought it could be used by another command (e.g. `yy` even though that works without referencing `vim::YankLine`). I can remove it if it's not needed.